link_apps.bash

```diff
@@ -0,0 +1,22 @@
+link_node_app () {
+    if ! [ -d $REBAR_BUILD_DIR/lib/$1 ]
+    then
+       ln -sdf $PWD/node/_build/dev1/lib/$1 $REBAR_BUILD_DIR/lib/$1;
```

Comment:
@radrow using `ln -d` is not possible on OS X because the `ln` command does not have a `-d` option.

As I understand from this [StackExchange comment](https://unix.stackexchange.com/a/267790) this option does not work on all filesystems.